flygym/examples/turning_controller.py

```diff
@@ -63,7 +63,10 @@ def __init__(
         **kwargs,
     ):
         # Check if we have the correct list of actuated joints
-        if fly.actuated_joints != all_leg_dofs:
+        if (
+            fly.actuated_joints != all_leg_dofs
+            and fly.actuated_joints != all_leg_dofs + ["joint_Head_yaw", "joint_Head"]
```

Comment:
Should this be just `if fly.actuated_joints != all_leg_dofs:` then, since the neck actuators are handled separately in a way that's not accessible to the user?